packages/dashboard/src/app/api/useStoryApi.js

```diff
@@ -127,33 +121,17 @@ const useStoryApi = (dataAdapter, { storyApi, encodeMarkup }) => {
         }
         isInitialFetch.current = false;
 
-        // Hardening in case data returned by the server is malformed.
-        // For example, story_data could be missing/empty.
-        const cleanStories = response.body.filter((story) =>
-          Array.isArray(story?.story_data?.pages)
-        );
+        const stories = response.body;
 
-        if (
-          cleanStories.length === 0 &&
-          cleanStories.length !== response.body.length
-        ) {
-          dispatch({
-            type: STORY_ACTION_TYPES.FETCH_STORIES_FAILURE,
-            payload: {
-              message: ERRORS.LOAD_STORIES.INCOMPLETE_DATA_MESSAGE,
```

Comment:
Let's also remove the actual `INCOMPLETE_DATA_MESSAGE` message as it's now unneeded.

Reply:
Done.